docs/_static/auth/iam.yaml

```diff
@@ -149,6 +153,52 @@ paths:
                 "$ref": "#/components/schemas/PresentationDefinition"
         "default":
           $ref: '../common/error_response.yaml'
+  "/iam/{id}/response":
+    post:
+      summary: Used by wallets to post the authorization response or error to.
+      description: | 
+        Specified by https://openid.net/specs/openid-4-verifiable-presentations-1_0.html#name-response-mode-direct_postjw
+        The response is either an error response with error, error_description and state filled or a submission with vp_token and presentation_submission filled.
```

Comment:
Why no state in success flow?

Reply:
In this PR the nonce is used to get the state. In the next PR it might just have the state for redirecting the error result